src/NuGet.Indexing/ILoader.cs

```diff
@@ -16,5 +16,7 @@ public interface ILoader
         /// <param name="config">Configuration to use.</param>
         /// <returns>Returns true if the data has changed.</returns>
         bool Reload(IndexingConfiguration config);
+
+        DateTime? GetLastUpdateTime(string name);
```

Comment:
For my own understanding, why is the file's last write time the key information here? Can't the reload method record when it latest returned true? This is more of an open question than a request for change.